wallet/storage/src/schema.rs

```diff
@@ -41,5 +41,7 @@ storage::decl_schema! {
         pub DBAddresses: Map<AccountDerivationPathId, Address>,
         /// Store for block/transaction entries
         pub DBTxs: Map<AccountWalletTxId, WalletTx>,
+        /// Store for wallet created transactions
+        pub DBUserTx: Map<AccountWalletCreatedTxId, SignedTransaction>,
```

Comment:
There is a minor inconsistency where `DBTxs` is plural while `DBUserTx` is singular.